src/hooks/useBalanceChartData/useBalanceChartData.ts

```diff
@@ -342,17 +344,24 @@ export const useBalanceChartData: UseBalanceChartData = args => {
   const priceHistoryDataLoading = useAppSelector(state =>
     selectPriceHistoriesLoadingByAssetTimeframe(state, assetIds, timeframe)
   )
+  const priceHistoryDataUnavailable = useAppSelector(state =>
```

Comment:
there's no need for this to be inside this hook - call it from the other view layer component

Reply:
This is for skip loading the balance chart if price history data is unavailable https://github.com/shapeshift/web/pull/1171/files#diff-e86a8d57bb214c8d7653fa226398d970f91eba7e09c1a2307d1a07a0320ca0a4R356